app/src/main/java/com/nowak/wjw/simplecompass/data/sensors/SensorHandler.java

```diff
@@ -0,0 +1,52 @@
+package com.nowak.wjw.simplecompass.data.sensors;
+
+import android.hardware.Sensor;
+import android.hardware.SensorEvent;
+import android.hardware.SensorEventListener;
+import android.hardware.SensorManager;
+
+import java.util.concurrent.TimeUnit;
+
+import io.reactivex.rxjava3.core.Observable;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import io.reactivex.rxjava3.subjects.BehaviorSubject;
+
+public class SensorHandler implements SensorEventListener {
+
+    private SensorManager mSensorManager;
+    private Sensor mVectorRotationSensor;
+    private BehaviorSubject<SensorEvent> eventProxy = BehaviorSubject.create();
+    //todo Q - is it public, private and expose a method, or just a method returning Observable?
+    public Observable<SensorEvent> mEventObservable = Observable
+            .interval(40, TimeUnit.MILLISECONDS) // 25 frames/second
```

Comment:
ktoś patrzy na to że to jest 40 ms - wystarczająca informacja, nie potrzeba komentarza.